src/commands/test.ts

```diff
@@ -1,12 +1,19 @@
 import * as path from 'path';
 import { spawn } from 'child_process';
 import { binPath } from '../utils';
+import { checkFileExists } from '../utils/checkFileExists';
 
 const defaultArgs = ['--watch'];
 
 export const test = async (args: string[] = defaultArgs) => {
   const jestBin = binPath('jest');
-  const configPath = path.resolve(__dirname, '../../jest.config.js');
+
+  const upperJestPath = '../../../../jest.config.js';
```

Comment:
Also it will be nice to have a util to resolve user (or upper) path

```
import {consumerPath} from '../utils/

const upperJestPath = consumerPath('jest.config.js');
```